src/site/content/en/lighthouse-accessibility/layout-table/index.md

```diff
@@ -1,32 +1,37 @@
 ---
 layout: post
-title: Ensure layout tables do not use data table elements
+title: "Presentational `<table>` elements do not avoid using `<th>`, `<caption>`, or the `[summary]` attribute"
 description: |
-  Learn about layout-table audit.
+  Learn how to make presentational tables more accessible for screen reader
+  users by avoiding child elements meant for data tables.
 date: 2019-05-02
+updated: 2019-09-19
 web_lighthouse:
   - layout-table
 ---
 
 Tables used for layout purposes shouldn't include data elements:
-`th` element, `caption` element or `summary` attribute.
-Lighthouse reports when layout `<table>` elements use `<th>`, `<caption>`, or `summary`.
+`<th>` element, `<caption>` element or `summary` attribute.
+
+## How this Lighthouse audit fails
+
+Lighthouse flags layout `<table>` elements that use `<th>`, `<caption>`, or `summary`:
 
 <!--
 ***Todo*** This audit doesn't seem to be failing for me. I added `role="presentation"` to a table
 with `th` expecting this to fail, and it does not.
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="layout-table.png" alt="Lighthouse audit showing presentational <table> elements incorrectly using <th>, <caption>, or summary
-  figcaption class with a value of 'w-figcaption'">
-    Presentational <code>&lt;table></code> elements incorrectly using <code>&lt;th></code>, <code>&lt;caption></code>, or <code>summary</code>.
-  </figcaption>
+  <img class="w-screenshot" src="layout-table.png" alt="Lighthouse audit showing presentational <table> elements incorrectly using <th>, <caption>, or [summary] attribute">
 </figure>
 -->
-## How to fix this problem
+
+{% include 'content/lighthouse-accessibility/scoring.njk' %}
+
+## How to make presentational tables accessible
 
 It's simple to pass the Lighthouse audit:
-remove data elements (`th`, `caption` elements or the `summary` attribute) from layout tables.
+remove data elements (`<th>`, `<caption>` elements or the `summary` attribute) from layout tables.
 
 If using a table just for visual layout, and not for tabular data,
```

Comment:
Maybe we should replace this section by telling folks to use CSS grid or Flexbox.

Reply:
I'm going to leave this for another PR where I take a pass at the tables and lists section as a whole.